ml-agents/mlagents/trainers/models.py

```diff
@@ -438,12 +438,15 @@ def create_observation_streams(
         h_size: int,
         num_layers: int,
         vis_encode_type: EncoderType = EncoderType.SIMPLE,
+        stream_scopes: List[str] = [],
```

Comment:
We shouldn't use default values of `[]` -- if two separate models are created using the defaults and one changes the value of `stream_scopes`, it will change for the other.

Reply:
Fixed, changed to None